pkg/runtime/podman/network.go

```
@@ -0,0 +1,120 @@
+	var merged *workspace.WorkspaceConfiguration
+
+	wsCfgLoader, err := config.NewConfig(filepath.Join(sourcePath, ".kaiden"))
+	if err == nil {
+		if wc, loadErr := wsCfgLoader.Load(); loadErr == nil {
+			merged = wc
+		}
+	}
+
+	projectLoader, err := config.NewProjectConfigLoader(storageDir)
+	if err != nil {
+		return merged, nil
```

golangci-lint (2.11.4)

[error] error is not nil (line 47) but it returns nil

(nilerr)